modules/Movecraft/src/main/resources/localisation/movecraftlang_en.properties

```diff
@@ -40,15 +39,14 @@ CrewSign\ -\ Craft\ Currently\ Piloted=You can't set your priority crew sign to
 CrewSign\ -\ Spawn\ Set=Priority crew bed set\!
 CrewSign\ -\ Respawn=Respawning at crew bed\!
 Cruise\ -\ Must\ Be\ Player=Only Players may cruise.
-Cruise\ -\ Craft\ Type\ Cannot\ Cruise=This craft type cannot cruise.
+Cruise\ -\ Craft\ Cannot\ Cruise=This craft type cannot cruise.
```

Comment:
I'd prefer if we changed the use of this to match the original key rather than changing the key

Reply:
Ok I will change it but this is not new though, both were there in the first place

Reply:
Requested changes made